filecoin/retriever.go

```diff
@@ -403,7 +430,7 @@ func (retriever *Retriever) lookupCandidates(ctx context.Context, cid cid.Cid) (
 		// Skip if we are currently at our maximum concurrent retrievals for this SP
 		// since we likely won't be able to retrieve from them at the moment even if
 		// query is successful
-		minerConfig := retriever.config.MinerConfig(candidate.MinerPeer.ID)
+		minerConfig := retriever.config.GetMinerConfig(candidate.MinerPeer.ID)
```

Comment:
no max concurrent without this because we don't set individual configs

Reply:
this does look like it was calling the method though, so concurrent config should be respected?

Reply:
yeah, sorry, a rush comment again, this one was correct out of all the others